packages/notebook-extension/src/index.ts

```diff
@@ -2512,7 +2518,14 @@ function addCommands(
         );
       }
     },
-    isEnabled,
+    isEnabled: args => {
+      const current = getCurrent(tracker, shell, args);
+      if (!current || !current.content.model) {
+        return false;
+      }
+      const length = current.content.model?.cells.length;
```

Comment:
Since you are checking for model higher up, you should be able to remove `?` here too I think. This might require you to assign `current.content.model` onto a variable before the check (or may not).

Reply:
Thanks for your review. I have tried to address most of your comments. Not sure to understand what you meant about assigning `current.content.model` onto a variable before the check.